bolt-design-system/bolt
# Video | Demo | Add demo of video triggering modal

## Jira

http://vjira2:8080/browse/BDS-1602

## Summary

Add demo of video triggering a modal to open on play.

## Details

Drupal implementation works better if video triggers modal to open, rather than modal triggering video to play. Add demo showing how this is done in Bolt.

## How to test

1. Run code locally
2. Go to [Video with Modal](http://localhost:3000/pattern-lab/patterns/02-components-video-70-video-with-modal/02-components-video-70-video-with-modal.html) page and verify that clicking on the "Play the video" button opens the plays the video in a modal and closing pauses the video.
